libs/architectures/mlpe/architectures/flows/flow.py

```diff
@@ -32,10 +50,16 @@ def linear_block(self) -> transforms.Transform:
     def distribution(self) -> distributions.Distribution:
         pass
 
+    @property
+    def context_dim(self):
+        sample_tensor = torch.zeros((1, self.n_ifos, self.strain_dim))
```

Comment:
Will this respect device change?

Reply:
Not sure. Haven't thought much about device logic, but is definitely necessary

Reply:
well sample_tensor is on the default cpu device. If the flow is on cuda, this is expected to break.

Reply:
Let me write some tests for this